src/library.js

```diff
@@ -4408,9 +4408,7 @@ LibraryManager.library = {
 #endif
                                "if (typeof dateNow !== 'undefined') {\n" +
                                "  _emscripten_get_now = dateNow;\n" +
-                               "} else if (typeof self === 'object' && self['performance'] && typeof self['performance']['now'] === 'function') {\n" +
-                               "  _emscripten_get_now = function() { return self['performance']['now'](); };\n" +
-                               "} else if (typeof performance === 'object' && typeof performance['now'] === 'function') {\n" +
+                               "} else if (typeof performance === 'object' && performance && typeof performance['now'] === 'function') {\n" +
```

Comment:
Can we drop the second clause?  (here and below)

Reply:
Unrelated to this change but why `performance['now']` and not simply `performance.now`?

Reply:
The second clause defends against `performance` being `null`. Debatable I guess how important that is, but I'd guessprograms having a local var called "performance" is rare but not super-rare. Anyhow, I figured it's best to not change that now.

The quotations are for closure compiler (which otherwise might minify it to `performance.a` etc.). However, it's possible recent closure does recognize that name, and the quotations are redundant (it would optimize them out though, so no harm).

Reply:
Ah I was assuming the first clause already protected against null..didn't realize null was an object.  nice.